src/integ/groovy/org/mastercoin/test/rpc/MSCSendToOwnersSpec.groovy

```diff
@@ -27,21 +27,32 @@ class MSCSendToOwnersSpec extends BaseRegTestSpec {
     def "STO calculates correct fees for the simple case"() {
         setup:
         def startingBTC = 10.0
-        def startingMSC = 1000
-        def amountSent = 100
+        def startingMSC = 1000.0
+        def amountSent = 100.0
         def fundedAddress = createFundedAddress(startingBTC, startingMSC)
         def currencyID = TMSC
+        def expectedBalance = 0.0
 
         when: "We Send to Owners"
-        def startingPropBal = getbalance_MP(fundedAddress, currencyID).balance
+        def startBalances = getallbalancesforid_MP(currencyID)
+        def startBalanceSender = getbalance_MP(fundedAddress, currencyID).balance
+        def numberOfHolders = startBalances.size()
         sendToOwnersMP(fundedAddress, currencyID, amountSent)
 
-        and: "we generate a block"
+        and: "We generate a block"
         generateBlock()
-        def balances = getallbalancesforid_MP(currencyID)
-        def numberDest = balances.size() - 1
-        def totalFee = numberDest * stoFeePerAddress
-        def expectedBalance = (numberDest == 0) ? startingPropBal : startingPropBal - amountSent - totalFee
+
+        // The fee for each receiver is #stoFeePerAddress
+        if (numberOfHolders > 1) {
```

Comment:
I don't know the spec or the implementation as well as you, but what I was thinking there might be a case where an address receives a SP that it didn't previously own in the same block that an STO was being made for that token. Wouldn't the behavior depend upon whether the transaction adding the SP to a new address occurred before or after the STO?

I realize we (likely) have control over this in RegTest mode so I'm thinking we could write tests to verify that this is being handled correctly in the Core implementation.